src/components/portal/portal.tsx

```diff
@@ -11,9 +11,10 @@ import { FlipModifier } from '@popperjs/core/lib/modifiers/flip';
  *
  * * Events might not bubble up as expected since the content is moved out to another DOM node
  * * Any styling that is applied to content from the parent will be lost, if the content is
- *   just another web compoent it will work without any issues
- * * When the node is moved in the DOM, `componentDidUnload`, `disconnectedCallback` and `connectedCallback`
- *   will be invoked, so `componentDidUnload` can not be used as a destructor (which is the wrong behavior anyway)
+ * just another web compoent it will work without any issues
+ * * When the node is moved in the DOM, `disconnectedCallback` and `connectedCallback`
+ * will be invoked, so if `disconnectedCallback` is used to do any tear-down, the
+ * appropriate setup will have to be done again on `connectedCallback`
```

Comment:
I updated this a little bit, to kind of reflect our current view on `disconnectedCallback`…